.gitignore

```diff
@@ -1 +1,2 @@
 /frontend
+/api
```

Comment:
Perhaps this should be `/*frontent` and `/*api`? The repo's aren't called `frontend` and `api`, but `jounrali-frontend` and `journali-api`.

Reply:
Journali is implied by the project name so 🤷‍♂